File: backend/src/routes/health.py
from flask import Blueprint, jsonify
from datetime import datetime

health_bp = Blueprint('health', __name__)

@health_bp.route('/health', methods=['GET'])
def health_check():
    """General health check endpoint"""
    return jsonify({
        'status': 'healthy',
        'service': 'tradepro-ai-backend',
        'timestamp': datetime.utcnow().isoformat(),
        'version': '1.0.0'
    }), 200

File: backend/src/routes/health.js
const express = require('express');
const router = express.Router();

// Health check endpoint
router.get('/', (req, res) => {
  const healthData = {
    status: 'healthy',
    timestamp: new Date().toISOString(),
    uptime: process.uptime(),
    memory: process.memoryUsage(),
    version: '1.0.0'
  };

  res.json(healthData);
});

module.exports = router;
